diff -up kernel/sound/soc/sunxi/sun8i-codec.c.omv~ kernel/sound/soc/sunxi/sun8i-codec.c
--- kernel/sound/soc/sunxi/sun8i-codec.c.omv~	2020-09-01 03:28:25.939748037 +0200
+++ kernel/sound/soc/sunxi/sun8i-codec.c	2020-09-01 03:30:10.318746034 +0200
@@ -175,9 +175,9 @@
 #define SUN8I_AIF3_CLK_CTRL_AIF3_CLOCK_SRC_MASK	GENMASK(1, 0)
 
 #define SUN8I_AIF_PCM_FMTS  (SNDRV_PCM_FMTBIT_S8|\
-			     SNDRV_PCM_FMTBIT_S16_LE|\
+			     SNDRV_PCM_FMTBIT_S16_LE/*|\
 			     SNDRV_PCM_FMTBIT_S20_LE|\
-			     SNDRV_PCM_FMTBIT_S24_LE)
+			     SNDRV_PCM_FMTBIT_S24_LE*/)
 #define SUN8I_AIF_PCM_RATES (SNDRV_PCM_RATE_8000_48000|\
 			     SNDRV_PCM_RATE_96000|\
 			     SNDRV_PCM_RATE_192000|\
