diff --git a/drivers/gpu/arm/mali400/mali/common/mali_executor.c b/drivers/gpu/arm/mali400/mali/common/mali_executor.c
index ee40520ed0ce7..0cf1ec0b0d028 100755
--- a/drivers/gpu/arm/mali400/mali/common/mali_executor.c
+++ b/drivers/gpu/arm/mali400/mali/common/mali_executor.c
@@ -1383,7 +1383,7 @@ static mali_bool mali_executor_is_suspended(void *data)
 	return ret;
 }
 
-static mali_bool mali_executor_is_working()
+static mali_bool mali_executor_is_working(void)
 {
 	MALI_DEBUG_ASSERT_EXECUTOR_LOCK_HELD();
 
