diff -up kernel/scripts/Makefile.lib.b11~ kernel/scripts/Makefile.lib
--- kernel/scripts/Makefile.lib.b11~	2021-04-25 00:28:04.228312941 +0200
+++ kernel/scripts/Makefile.lib	2021-04-25 00:29:10.879316238 +0200
@@ -328,7 +328,7 @@ $(obj)/%.dtb.S: $(obj)/%.dtb FORCE
 
 quiet_cmd_dtc = DTC     $@
 cmd_dtc = $(HOSTCC) -E $(dtc_cpp_flags) -x assembler-with-cpp -o $(dtc-tmp) $< ; \
-	$(DTC) -o $@ -b 0 \
+	$(DTC) -@ -o $@ -b 0 \
 		$(addprefix -i,$(dir $<) $(DTC_INCLUDE)) $(DTC_FLAGS) \
 		-d $(depfile).dtc.tmp $(dtc-tmp) ; \
 	cat $(depfile).pre.tmp $(depfile).dtc.tmp > $(depfile)
